superset/__init__.py

```diff
@@ -149,12 +150,16 @@ def index(self):
         return redirect('/superset/welcome')
 
 
+custom_sm = app.config.get('CUSTOM_SECURITY_MANAGER')
+if not custom_sm:
+    custom_sm = SupersetSecurityManager
+
```

Comment:
We should absolutely raise with an explicit message with perhaps a link to the UPDATE notes `if not issubclass(custom_sm, SupersetSecurityManager)`